### PR TITLE
Fredriks completed week3-assessment 

### PR DESCRIPTION
**Pull request by fredrikfalkman**

### CODE REVIEW

File: spec/user_spec.rb [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
+  before do
+    subject = User.new
+    subject.name = "Amber"
+    subject.save
+  end
+
   it 'can create a new user' do
-    user = User.create
-    expect(user).not_to be nil
+    expect(User.last).not_to be nil
```
Comment (AmberWilkie) [inline spec/user_spec.rb:12]: Excellent. Noticed that I wrote this test and it doesn't do anything, so nice catch.

File: features/step_definitions/basic_steps.rb [hunk 2]
```diff
@@ -1 +1,19 @@
...
+  visit "/name/#{param_name}"
+end
+
+Then(/^I should create a new user with name "([^"]*)"$/) do |name|
+  expect(User.last.name).to eq name
+end
+
+Then(/^I should see "([^"]*)"$/) do |answer|
+  expect(page).to have_content answer
+end
```
Comment (AmberWilkie) [inline features/step_definitions/basic_steps.rb:19]: This is the exact same step as the `I see` step above on line 5. You can and should combine them to keep your code DRY.

**Review by AmberWilkie — APPROVED** [review]
Really great work. I'm very happy.

**Comment by fredrikfalkman** [comment]
Thanks!

**Pull request closed without merging**